style.css

```diff
@@ -341,11 +337,12 @@ input {
 
 .video-fileinput-img {
     position: absolute;
-    z-index: 10;
+    z-index: 6;
     display: none;
-    top: 50%;
-    transform: translateY(-50%);
-    max-width: 100%;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    border: 6px solid var(--btnBg);
```

Comment:
Why such a big border? Is it in the original design?

Reply:
Yeah so the border doesn't fix the glitch, I just added it preemptively as it was missing based on https://github.com/bdebon/youtube-thumbnail-tester-chrome-extension/issues/27

Reply:
Ah you're 100% right!
Then thank you for all the good work! I'm gonna proudly merge it!